tsconfig-build.json

```diff
@@ -4,7 +4,11 @@
     "declaration": true,
     "noEmit": false,
     "rootDir": "src/",
-    "emitDeclarationOnly": true
+    "emitDeclarationOnly": true,
+    "types": [
```

Comment:
comment: this file already extends the `tsconfig.json` so we shouldn't need to define the `types` again. Also I'm not sure why we'd need to define them for the build as test files and jest isn't part of the distributed files